src/microbit/model/constants.py

```diff
@@ -0,0 +1,16 @@
+BLANK = "00000:00000:00000:00000:00000:"
+BOAT = "05050:05050:05050:99999:09990:"

```

Comment:
Also maybe we should have another file for all these image constants because this file will get cluttered very quickly with all the images.

Reply:
For now we are keeping these here since we don't know how we're going to access all of the shapes (it can be like the letters where they are parsed from a larger array).